charts/stable/reg/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: v2
+appVersion: 0.16.1
+description: Docker registry v2 command line client and repo listing generator with security checks.
+name: reg
+version: 1.0.0
+kubeVersion: ">=1.16.0-0"
+keywords:
+- reg
+home: https://github.com/k8s-at-home/charts/tree/master/charts/stable/reg
+sources:
```

Comment:
It's normally a good practice to also include some sort of icon. Maybe there is some sort of Github avatar for `genuinetools` that can be "borrowed"?

Reply:
I can take a look. I know that there isn't one for the app.